arkouda/groupbyclass.py

```diff
@@ -338,9 +338,10 @@ def count(self) -> Tuple[groupable, pdarray]:
         >>> counts
         array([1, 2, 4, 3])
         """
-        cmd = "countReduction"
-        args = "{} {}".format(cast(pdarray, self.segments).name, self.length)
-        repMsg = generic_msg(cmd=cmd, args=args)
+        repMsg = generic_msg(
+            cmd="countReduction",
+            args={"segments": cast(pdarray, self.segments).name, "size": self.length},
```

Comment:
It's not hurting anything, but the `.name` for `segments` is unnecessary right?

Reply:
If I remember correctly, there was an issue with `Datetime` because it wasn't yet added to the JSON parsing. Since that got merged, I will look into updating this again.

Reply:
Removed the name. I also had to add JSON parsing for `Timedelta` as well.